Flaky test TestReplicationWithLimitSpeed

```
--- FAIL: TestReplicationWithLimitSpeed (10.10s)
    client.go:72: 
        	Error Trace:	/Users/runner/work/incubator-kvrocks/incubator-kvrocks/tests/gocase/integration/replication/client.go:72
        	            				/Users/runner/work/incubator-kvrocks/incubator-kvrocks/tests/gocase/integration/replication/replication_test.go:209
        	Error:      	Received unexpected error:
        	            	i/o timeout
        	Test:       	TestReplicationWithLimitSpeed
```

Unstable test. @git-hulk perhaps release the read timeout for the specific case.

_Originally posted by @tisonkun in https://github.com/apache/incubator-kvrocks/issues/998#issuecomment-1279942166_

Check run: https://github.com/apache/incubator-kvrocks/actions/runs/3258497049/jobs/5350638195

--- a/tests/gocase/integration/replication/replication_test.go
+++ b/tests/gocase/integration/replication/replication_test.go
@@ -205,7 +205,7 @@ func TestReplicationWithLimitSpeed(t *testing.T) {
 	})
 	defer master.Close()
 	masterClient := master.NewClientWithOption(&redis.Options{
-		ReadTimeout:  10 * time.Second,
+		ReadTimeout: 10 * time.Second,
 	})
 	defer func() { require.NoError(t, masterClient.Close()) }()
 	util.Populate(t, masterClient, "", 1024, 10240)

--- a/tests/gocase/integration/replication/replication_test.go
+++ b/tests/gocase/integration/replication/replication_test.go
@@ -206,7 +206,6 @@ func TestReplicationWithLimitSpeed(t *testing.T) {
 	defer master.Close()
 	masterClient := master.NewClientWithOption(&redis.Options{
 		ReadTimeout:  10 * time.Second,
-		WriteTimeout: 10 * time.Second,
 	})
 	defer func() { require.NoError(t, masterClient.Close()) }()
 	util.Populate(t, masterClient, "", 1024, 10240)

--- a/tests/gocase/integration/replication/replication_test.go
+++ b/tests/gocase/integration/replication/replication_test.go
@@ -204,7 +204,10 @@ func TestReplicationWithLimitSpeed(t *testing.T) {
 		"rocksdb.target_file_size_base": "1",
 	})
 	defer master.Close()
-	masterClient := master.NewClient()
+	masterClient := master.NewClientWithOption(&redis.Options{
+		ReadTimeout:  10 * time.Second,
+		WriteTimeout: 10 * time.Second,
+	})
 	defer func() { require.NoError(t, masterClient.Close()) }()
 	util.Populate(t, masterClient, "", 1024, 10240)
 

